app/controllers/players_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PlayersController < ApplicationController
+  def create
+    redirect_to new_user_session_path unless user_signed_in?
```

Comment:
This statement doesn't work... which makes no sense because it works perfectly fine in the other controller (instances#create) I have the feeling this is an issue with form_for, but I have no idea how to change it. I would suggest that we proceed as the code works when we are logged in - and I'll grab a teacher on Tuesday to ask for help

Reply:
Ok, also created a Trello for this so we don't lose track: https://trello.com/c/N0Pssf25/24-dbi-after-clicking-create-redirect-to-login-page-if-user-is-not-logged-in